Userland/Libraries/LibC/inttypes.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2018-2022, the SerenityOS developers.
```

Comment:
Do we have a history of "merging" Andreas into "the SerenityOS developers"? Or should we just append it to the list instead?

Reply:
That's actually correct, I thought we had done that but I can't find any instance.